crates/uv/src/lib.rs

```diff
@@ -1141,6 +1141,12 @@ async fn run_project(
                     .combine(Refresh::from(args.settings.reinstall.clone()))
                     .combine(Refresh::from(args.settings.upgrade.clone())),
             );
+            // If the target is a PEP 723 script, parse it.
```

Comment:
I think we should consider moving this up to where we parse the script for `uv run`, nearer to the top of the file.

Reply:
Done